drivers/espi/espi_mchp_xec.c

```diff
@@ -21,6 +21,10 @@
  */
 #define ESPI_XEC_VWIRE_SEND_TIMEOUT 100ul
 
+#define VW_MAX_GIRQS                2ul
+/* Missing HAL value */
+#define MEC_ESPI_MSVW08_SRC1_VAL    5ul
```

Comment:
@scottwcpg this is the missing HAL value. I have updated the PR with device tree changes